packages/http-client-python/emitter/src/lib.ts

```diff
@@ -56,6 +56,27 @@ const EmitterOptionsSchema: JSONSchemaType<PythonEmitterOptions> = {
 const libDef = {
   name: "@typespec/http-client-python",
   diagnostics: {
+    // error
+    "unknown-error": {
+      severity: "error",
+      messages: {
+        default: paramMessage`Can't generate Python client code from this TypeSpec, please open an issue on https://github.com/microsoft/typespec'.\n${"stack"}`,
+      },
+    },
+    "invalid-models-mode": {
+      severity: "error",
+      messages: {
+        default: paramMessage`Invalid value '${"inValidValue"}' for 'models-mode' of tspconfig.yaml and expected values are 'dpg'/'none'.`,
+      },
+    },
+    "no-python-installed": {
+      severity: "error",
+      messages: {
+        default:
+          "Python is not installed. Please follow https://www.python.org/ to install Python or set 'use-pyodide' to true.",
```

Comment:
this shouldn't be an error, because we should automatically call `use-pyodide` if that is the case. The only way an error like this would occur is if someone explicitly set `use-pyodide=false` without having python. In that case, I think the error name would be better being something like `pyodide-flag-conflict` or something like that

Reply:
Make sense and I think `pyodide-flag-conflict` is reasonable here.
Updated.

Reply:
can you make `invalid-models-mode` and potentially `pyodide-flag-conflict` have severity `warning` instead of `error`? We can continue generating in these situations, we just have to fallback. We really should only throw errors if there is no way for us to continue. For `invalid-models-mode`, we can just generat without models. For `pyodide-flag-conflict`, one option is to just generate with `pyodide`, but IU'm also not sure that's the best solution since the user explicitly said `use-pyodide=false`. what do you guys think?

Reply:
If users already set explicit value, they may want to control emitter for specific behavior. Based on the assumption, I think we had better not fall back